src/component/Timer/index.jsx

```diff
@@ -22,11 +31,43 @@ const Timer = () => {
 
     const reset = () => {
       timeOverSoundAudio.play();
+      const showNotification = () => {
+        let text = null;
+        const img = 'https://source.unsplash.com/1600x900/?nature,water;';
+        if (counter.current === 6) {
```

Comment:
Why `6`? Shouldn't it be `7`? Just like it is defined here:

https://github.com/the-sol/pomodoro/blob/main/src/services/timer.js#L36-L38

Reply:
because the counter start calculates from 0, not 1, and since we need to show a long break after 7 periods, it must be 6, not 7.

Reply:
You should not even rely on the `counter` to determine which message to show. You should rely on the `nextPeriod.id` and `currentPeriod.id` instead since that's what you care about.